README.md

```diff
@@ -103,13 +103,13 @@ All modules are disabled by default and need to be activated explicitly in your
 
 ```lua
 require'nvim-treesitter.configs'.setup {
-  -- One of "all", "maintained" (parsers with maintainers), or a list of languages
-  ensure_installed = "maintained",
+  -- One of "all", "maintained" (deprecated), or a list of languages
```

Comment:
I would not document deprecated stuff.

Reply:
The idea was to sprinkle the deprecation warning liberally throughout the docs to increase the chances of people seeing it...